proto/osmosis/concentrated-liquidity/tx.proto

```diff
@@ -132,4 +132,57 @@ message MsgCollectIncentivesResponse {
     (gogoproto.moretags) = "yaml:\"collected_incentives\"",
     (gogoproto.nullable) = false
   ];
+}
+
+// ===================== MsgCreateIncentive
+message MsgCreateIncentive {
+  uint64 pool_id = 1 [ (gogoproto.moretags) = "yaml:\"pool_id\"" ];
+  string sender = 2 [ (gogoproto.moretags) = "yaml:\"sender\"" ];
+  string incentive_denom = 3;
+  string incentive_amount = 4 [
```

Comment:
just a thought, why not set this as Coin instead of denom and amount? so that later down the line we can create incentives for many Coins

Reply:
Agreed on making this a Coin, unless it was by design for a reason I am not seeing.